packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -148,13 +148,12 @@ export const Headline = styled.h3`
 
     return topStory ? getParagon(script) : getPica(script);
   }}
-
   color: ${C_EBON};
   ${({ service }) => getSerifMedium(service)}
   margin: 0; /* Reset */
   padding-bottom: ${GEL_SPACING};
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    ${props => (props.script ? getGreatPrimer(props.script) : '')};
+    ${({ script }) => script && getParagon(script)};
```

Comment:
Since you are destructuring the `script` prop here, could you also update the other ones to use the same technique?

Reply:
Reverted to not desctructuring and opened https://github.com/bbc/psammead/issues/1921